.gitignore

```diff
@@ -5,3 +5,4 @@ db/*.sqlite3
 db/schema.rb
 db/data.yml
 config/memcache.conf
+.ruby-gemset
```

Comment:
Enough people coming to the project use gemsets that this is probably ok, but if I were you I'd set up a global gitignore in your git config and then you don't have to add things that aren't really project specific to every project you work on.